### PR TITLE
Added multiple social buttons ideas

### PR DESCRIPTION
**Pull request by Tushar11Raj**
Added more ideas in social media button

Added new ideas to add pinterest, behance and youtube buttons under social media category.

Issue: New button ideas

### CODE REVIEW

File: BUTTON_IDEAS.md [hunk 1]
```diff
@@ -61,5 +61,11 @@ When adding ideas, please add them in the right category (or add a new one if yo
 
 | status  | Button name                    | Description                          | Submitted by                                  |
 | ------- | ------------------------------ | ------------------------------------ | --------------------------------------------- |
+
```
Comment (shahednasser) [inline BUTTON_IDEAS.md:64]: Remove this extra spacing as it ruins the table's format

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: BUTTON_IDEAS.md (hunk 1 above)
Comment (Tushar11Raj) [reply BUTTON_IDEAS.md:64]: ok sure

**Comment by Tushar11Raj** [comment]
Review it once. Changes have been done

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @Tushar11Raj for idea

**Comment by Tushar11Raj** [comment]
@shahednasser Can you please add hacktoberfest-accepted tag to it.

**Comment by shahednasser** [comment]
@Tushar11Raj that is not necessary. As long as this PR does not have the invalid label it will count.

**Pull request merged**